src/component/MembersList.jsx

```diff
@@ -1,10 +1,14 @@
 import React, { useState, useEffect } from "react";
-import Member from "./Member";
+import { useParams } from "react-router-dom";
+import Member from "../pages/Member";
 import { apiPath } from '../api';
+import '../styles/memberList.css';
 
 const MembersList = ({teamId}) => {
   const [members, setMembers] = useState([]);
   const [loading, setLoading] = useState(true);
+  const [team, setTeam] = useState(null);
+  const { code } = useParams();
 
   useEffect(() => {

```

Comment:
as we fetch data for both team and member in the useEffect, try to use promise.all if possible to improve effeciency. something like

```js
const [teamData, membersData] = await Promise.all([
          fetchTeam(),
          fetchMembers()
        ]);
```